src/XmlUtility.ps1

```diff
@@ -0,0 +1,128 @@
+﻿$WikiConfig = @{
+    CacheDir = Resolve-FullPath $Config.ArtifactsDir 'cache'
+    WikiDir = Resolve-FullPath $Config.RootDir '..\..\SpbDotNet.wiki'
+}
+
+$WikiRepository = @{
+    Communities = @{}
+    Meetups = @{}
+    Talks = @{}
+    Speakers = @{}
+    Friends = @{}
+    Venues = @{}
```

Comment:
You don't need a dictionaries. You have enough lists.

Reply:
Fixed